lib/raygun/client.rb

```diff
@@ -141,6 +141,8 @@ def build_payload_hash(exception_instance, env = {})
         tags = env.delete(:tags) || []
         tags << rack_env if rack_env_present?
 
+        groupingKey = env.delete(:groupingKey)
```

Comment:
Could you use `grouping_key` for this in the `ENV` hash and the local variable? We only want to change to `groupingKey` when we add it to the hash.